pkg/ee/default-application-catalog/applicationdefinitions/argocd-app.yaml

```diff
@@ -38,3 +38,106 @@ spec:
       service:
         # To Expose ArgoCD externally without ingress, set service type as "LoadBalancer". Default value is "ClusterIP".
         type: "LoadBalancer"
+  documentationURL: https://argoproj.github.io/cd/
```

Comment:
Why did you make all the YAML files executable?

![2024-02-19T14-03-24](https://github.com/kubermatic/kubermatic/assets/127499/bc808f5a-c7da-4092-9fc3-7880557f632a)

chmod 644 was good, why make it 755?